.github/workflows/mkdocs.yaml

```diff
@@ -0,0 +1,44 @@
+on:
+  push:
+    branches:
+      - main
+      - master
+
+name: Make Sphinx API Docs
+
+jobs:
+  mkdocs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: s-weigand/setup-conda@v1
+      - name: Build Sources and Docs
+        run: |
+          source bin/install_travis.sh
```

Comment:
This should happen after the one for C++ has been run. i.e. after line 23

Reply:
Yup, done.